lib/x11utils.pm

```diff
@@ -322,4 +324,20 @@ sub untick_welcome_on_next_startup {
     }
 }
 
+=head2 workaround_bsc1169203
+
+ workaround_bsc1169203
+
+Bad performance on Wayland/GNOME causes opensuse-welcome popup to fail.
+
+=cut
+sub workaround_bsc1169203 {
+    if (check_screen 'opensuse-welcome-started-not-shown-bsc1169203') {
```

Comment:
As we already use `match_has_tag`  when calling this method, I believe we can get rid of this `check_screen`

Reply:
No, it is necessary because there are three different failures:

1. opensuse-welcome starts but window is not shown
2. opensuse-welcome crashes and is not startet
3. opensuse-welcome causes Wayland/GNOME login to crash

The if differentiates between 1. and 2., for 3. ATM it has to fail. The workaround would be to select X11/Gnome from GDM in that case.

Reply:
Awesome, there is a case 4., opensuse-welcome fails to open until timeout: http://openqa.slindomansilla-vm.qa.suse.de/tests/2460